scanner/image/db/scripts/docker-entrypoint.sh

```diff
@@ -353,7 +359,11 @@ _main() {
 		fi
 	fi
 
-	exec "$@"
+	### STACKROX MODIFIED - Start Postgres as a child process.
+	"$@" &
```

Comment:
You'll also probably want to the add the line above about flock from central-db

Reply:
also in case you didn't see this: https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes

Seems like `ReadWriteOncePod` may also solve this problem instead of `flock`

Reply:
This is beta in 1.27 so we can't rely on it